src/actions/market.js

```diff
@@ -5,22 +5,18 @@ import uuid from 'uuid/v4'
 import * as api from 'api'
 
 import { receiveEntities, updateEntity } from 'actions/entities'
-
+import { openModal, closeModal } from 'actions/modal'
+import { refreshTokenBalance } from 'actions/blockchain'
```

Comment:
There is something that I do not understand. If you take a look into the ```refreshTokenBalance```  function, you will notice a couple of things. 

First I was not able to find in ```src/actions/blockchain``` in ```development``` or ```backmerge/redo```  branches.  

The second, that function seems to be created for Olympia purposes, do not know if that function should be in management or not. 

My guess is we have to remove it because we are getting the balance from the provider and we do not have a collateral token.

Reply:
👍